hungarian/malyvacukor.md

```diff
@@ -1,9 +1,11 @@
-Egy szep reggel meztelenul ebredsz egy malyvacukor agyban, malyvacukor szobaban.
+Egy szép reggel meztelenül ébredsz egy mályvacukor ágyban, mályvacukor szobában.
 
-Mit csinalsz ?
+Mit csinálsz?
 
 [visszaalszol](alvas/alom.md)
 
 [felveszel egy par malyva zoknit] (zokni/zokni.md)
 
-[lezuhanyzol](zuhanyzas/zuhanyzas.md)
\ No newline at end of file
+[lezuhanyzol](zuhanyzas/zuhanyzas.md)
+
+[megreggelizel](reggeli/reggeli.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.